### PR TITLE
Added support for Sanskrit language

### PR DESCRIPTION
**Pull request by snsten**
Added initial Sanskrit language support with:

- stop_words
- lex_attrs
- examples

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/lang/sa/examples.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+
+"""
+Example sentences to test spaCy and its language models.
+
+>>> from spacy.lang.hi.examples import sentences
```
Comment (svlandeg) [inline spacy/lang/sa/examples.py:8]:
```suggestion
>>> from spacy.lang.sa.examples import sentences
```

File: spacy/tests/lang/sa/test_text.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@pytest.mark.parametrize(
+    "text,length",
+    [
+        ("श्री भगवानुवाच पश्य मे पार्थ रूपाणि शतशोऽथ सहस्रशः।", 9,),
+        ("गुणान् सर्वान् स्वभावो मूर्ध्नि वर्तते ।", 6),
+    ],
+)
+def test_sa_tokenizer_handles_cnts(sa_tokenizer, text, length):
+    tokens = sa_tokenizer(text)
+    assert len(tokens) == length
```
Comment (svlandeg) [inline spacy/tests/lang/sa/test_text.py:22]: Perhaps you could add a test here for the `like_num` functionality as well? See https://github.com/explosion/spaCy/blob/master/spacy/tests/lang/es/test_text.py#L56 for inspiration - you could keep most of it, but swap the Spanish words for relevant Sanskrit ones. Having a few `False` cases is always good, too.

**Review by svlandeg — COMMENTED** [review]
This looks great, thanks so much for your contribution! Also really appreciate adding unit tests right away, and creating the `sa_tokenizer` fixture :-)

I just had 2 small comments, but other than that we can definitely merge this, and spaCy will speak Sanskrit!

**Comment by kleczekr** [comment]
Hi! Is anyone working on the Sanskrit tools in the library?

**Comment by svlandeg** [comment]
Hi @kleczekr, you'd be best off asking this question in the [discussions forum](https://github.com/explosion/spaCy/discussions)!

**Pull request merged**